pkg/operator/clustermemberremovalcontroller/clustermemberremovalcontroller.go

```diff
@@ -125,15 +125,23 @@ func (c *clusterMemberRemovalController) attemptToScaleDown(ctx context.Context,
 	if desiredControlPlaneReplicasCount == 0 {
 		return fmt.Errorf("desired control plane replicas count cannot be empty")
 	}
-	if currentVotingMemberIPListSet.Len() <= desiredControlPlaneReplicasCount {
-		klog.V(4).Infof("Ignoring scale-down since the number of etcd voting members (%d) < desired number of control-plane replicas (%d) ", currentVotingMemberIPListSet.Len(), desiredControlPlaneReplicasCount)
+
+	liveVotingMembers, err := c.getAllVotingMembers(ctx)
+	if err != nil {
+		// TODO : update status condition
+		// TODO : should go degraded ?
```

Comment:
nope, that's a transient issue and will be retried next recon cycle

Reply:
when you want to tell the admin to fix something specifically. I'd personally still prefer logs as they are a bit easier to understand over time. 

Going Degraded only if we should block the upgrades, but my philosophy is also different than what's used across the codebase.